index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css">
+  <title>Portfolio - Setup and Mobile First</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="toolbar">
+    <div class="container-2">
+        <div class="logo"><a href="#home"><p class="logo-txt poppins">My Logo</p></a></div>
+        <nav class="menu"><img class="menu-im" alt="Menu" width="24px" height="24px" src="images/menu.svg"></nav>
+    </div>
+  </header>
+  <section class="headline poppins">
+    <h1 class="name">I'm Juan Carlos. Glad to see you!</h1>
+    <p class="desc">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    <p class="connect">LET'S CONNECT</p>
```

Comment:
- [ ] please kindly make this an `hx(2/3)` tag here as it is a subtitle. This way will be semantically correct and the screen readers and other technologies will thank you